frontend/public/components/graphs/base.jsx

```diff
@@ -149,6 +151,7 @@ BaseGraph.propTypes = {
       })),
   ]),
   percent: PropTypes.number, // for gauge charts
+  wrapperClass: PropTypes.string,
```

Comment:
Just call this `className` to be consistent with other components that take a similar property.

I'm confused why this is needed now and wasn't before.

Reply:
I'm passing in `--compact` classes which shrinks the gauge charts in the dashboard, as opposed to on the detail page.  It is the `chartClass` in the example below:
```
<QuotaGaugeCharts quota={quota} resourceTypes={resourceTypes} 
  rowClass="co-resource-quota-chart-row--compact" 
  chartClass="graph-wrapper--compact" />
```
I could pass in a single `compact` flag like I'm doing for the scopes on the dashboard:
```
<QuotaScopes scopes={scopes} compact />
```
Probably makes more sense to do it like ^

Reply:
fixed